function/colcon_cd.sh

```diff
@@ -1,7 +1,7 @@
 # copied from colcon-cd/function/colcon_cd.sh
 
 colcon_cd() {
-  if [ "$_ARGCOMPLETE" == "1" ]; then
+  if [ "$_ARGCOMPLETE" = "1" ]; then
```

Comment:
This appears to correct a simple typo in the original PR's intended string comparison.